docs/user/versions.rst

```diff
@@ -1,132 +1,64 @@
-If you develop on a branch that is different than the default for your VCS,
-you should set the **Default Branch** to that branch.
-
-You should push a **tag** for each version of your project.
-These tags should be numbered in a way that is consistent with semantic versioning.
-This will map to your ``stable`` branch by default.
-
-.. note::
-    We in fact are parsing your tag names against the rules given by
-    `PEP 440`_. This spec allows "normal" version numbers like ``1.4.2`` as
-    well as pre-releases. An alpha version or a release candidate are examples
-    of pre-releases and they look like this: ``2.0a1``.
-
-    We only consider non pre-releases for the ``stable`` version of your
-    documentation.
-
-If you have documentation changes on a **long-lived branch**,
-you can build those too.
-This will allow you to see how the new docs will be built in this branch of the code.
-Generally you won't have more than 1 active branch over a long period of time.
-The main exception here would be **release branches**,
-which are branches that are maintained over time for a specific release number.
-
-.. _PEP 440: https://www.python.org/dev/peps/pep-0440/
-
 Version states
 --------------
 
-States define the visibility of a version across the site.
-You can change the states of a version from the :guilabel:`Versions` tab of your project.
-
-Active
-~~~~~~
+Each version of your documentation has a combination of three states (**Active**, **Public**, and **Hidden**) which determine its visibility on your site:
 
-- **Active**
+You can change the states for each version of your documentation in the :guilabel:`Versions` tab of your project.
 
-  - Docs for this version are visible
-  - Builds can be triggered for this version
+**Active** or **Inactive**
+  - **Active** docs are visible, and builds can be triggered for the documentation.
+  - Docs for **Inactive** versions *are deleted* and builds cannot be triggered.
 
-- **Inactive**
+**Hidden** or **Not hidden**
+  - **Hidden** docs are not listed on the :term:`flyout menu` on the docs site,
+    or shown in search results from another version on the docs site
+    (like on search results from a superproject).
+  - **Not hidden** docs are listed on the :term:`flyout menu` on the docs site,
+    and are shown in search results on the docs site.
 
-  - Docs for this version aren't visible
-  - Builds can't be triggered for this version
+  Hiding a version doesn't make it private,
+  any user with a link to its docs can still see it.
+  This is useful when:
 
-When you deactivate a version, its docs are removed.
+  - You no longer support a version, but you don't want to remove its docs.
+  - You have a work in progress version and don't want to publish its docs just yet.
 
-Hidden
-~~~~~~
+  Hidden verions are listed as ``Disallow: /path/to/version/``
+  in the default `robots.txt file <https://www.robotstxt.org/>`__ created by Read the Docs.
```

Comment:
We should probably link to our robots documentation page here instead: https://docs.readthedocs.io/en/stable/reference/robots.html